scripts/bump-year.js

```diff
@@ -17,21 +17,22 @@ git commit -am "Internal: Bumped the year." && git push
 
 */
 
-require( '@ckeditor/ckeditor5-dev-bump-year' )
-	.bumpYear( {
-		cwd: process.cwd(),
-		globPatterns: [
-			{ // LICENSE.md, .eslintrc.js, etc.
-				pattern: '*',
-				options: {
-					dot: true
-				}
-			},
-			{
-				pattern: '.husky/*'
-			},
-			{
-				pattern: '!(coverage|.nyc_output|dist|demo)/**'
+import { bumpYear } from '@ckeditor/ckeditor5-dev-bump-year';
+
+bumpYear( {
+	cwd: process.cwd(),
+	globPatterns: [
+		{ // LICENSE.md, .eslintrc.js, etc.
+			pattern: '*',
+			options: {
+				dot: true
 			}
-		]
-	} );
+		},
+		{
+			pattern: '.husky/*'
+		},
+		{
+			pattern: '!(coverage|.nyc_output|dist|demo)/**'
```

Comment:
`.nyc_output` will not be generated anymore.
`demo` does not exist now.

Reply:
The `demo` folder exists again, as I moved `src/demo/*` to `/demo/*`.

The `.nyc_output` folder doesn't exist and didn't exist in the old version, but it's created during the CI (see https://github.com/ckeditor/ckeditor5-vue/blob/master/.circleci/config.yml#L76-L88), so I think we should keep it, just in case.